src/adapters/github.less

```diff
@@ -353,6 +353,11 @@
           height: 60px;
         }
 
+        .octotree-disclaimer-token {
```

Comment:
Why add a new selector? We already have .octotree-disclaimer. Can we use it (maybe customer so it is usable in both places?